docs/2.8.0/docs/kubernetes/01-introduction/overview.rst

```diff
@@ -7,57 +7,57 @@ Deploy on Kubernetes
 Audience
 ********
 
-This content is targeted primarily for individuals with the following skills & experience:
+This content targets individuals with the following skills and experience:
 
 * Basic hands-on experience with deploying and managing applications running on `Kubernetes <https://kubernetes.io/>`_.
 * Deploying applications packaged as `Helm charts <https://helm.sh/>`_.
 * Strong understanding of operational concepts of the chosen cloud provider, particularly monitoring and observability.
-* Basic know-how of deployment and configuration of the chosen cloud provider's components to adhere to tight security requirements, including setting up an IDP provider (Auth0, Azure AD, etc.), certificate issuer (GCP CA Service, AWS Private CA, etc.).
+* Basic know-how of deployment and configuration of the chosen cloud provider's components to adhere to tight security requirements. This includes setting up an IDP provider (Auth0, Azure AD, and so on), and a certificate issuer (such as GCP CA Service, AWS Private CA).
 
-Content Covered
+Content covered
```

Comment:
Yes, that was previously the convention. However, now we are using the [Google developer guide](https://developers.google.com/style/headings) for docs.daml.com, which calls for sentence case in headings. At some point, we would like to change all the headings, but it will take awhile. In the meantime, we are changing subheadings to sentence case as we come across them and temporarily leaving the main headings in title case to match other headings in the table of contents.